create_campaign/lambda_function.py

```diff
@@ -0,0 +1,86 @@
+import json
+import boto3
+import uuid
+from datetime import datetime, timedelta
+import logging
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
```

Comment:
這邊的 Logger 設定錯誤，這樣應該印不出來

```suggestion
logger = logging.getLogger()
logger.setLevel(logging.INFO) # or logger.setLevel("INFO")
```

官方文檔: https://docs.aws.amazon.com/lambda/latest/dg/python-logging.html